test-data/src/main/java/dev/futa/tutorial/pesel/junit/AssertOptional.java

```diff
@@ -0,0 +1,19 @@
+package dev.futa.tutorial.pesel.junit;
+
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class AssertOptional {
```

Comment:
Własne asercje to za dużo, ale ok, bawmy się :-)

Reply:
właśnie to robię i wiesz jak radocha?